synfig-studio/src/gui/dialogs/dialog_setup.h

```diff
@@ -99,6 +99,7 @@ class Dialog_Setup : public Dialog_Template
 	void on_def_background_type_changed(); //bound on clicked
 	void on_def_background_color_changed();
 	void on_def_background_image_set();
+	void on_number_of_core_select();
```

Comment:
Looking to other signals slots/callbacks name: prefer `on_number_of_core_changed()`.
In earlier commits you used `on_core_changed()`, and it led to a confusion: did the core changed? No, the number of core (or threads).

Reply:
is `on_number_of_thread_changed()`  fine

Reply:
Agree. But better on_number_of_thread*s*_changed()